tests/installation/bootloader.pm

```diff
@@ -19,6 +19,12 @@ use registration;
 # hint: press shift-f10 trice for highest debug level
 sub run() {
     my ($self) = @_;
+    # arbitrary slow typing speed, also see bootloader_uefi
+    my $slow_typing_speed = 13;
+    # type even slower towards the end to ensure no keybuffer overflow even
+    # when scrolling within the boot command line to prevent character
+    # mangling
+    my $slower_typing_speed = 4;
```

Comment:
slow is faster than slower? (slow(13), slower(4), slowest(?)

Just to note: I see many more failures on 'bootloader' in Staging now than I used to...